specification/fist/resource-manager/readme.java.md

```diff
@@ -0,0 +1,7 @@
+## Java
+
+These settings apply only when `--java` is specified on the command line.
+
+```yaml $(java)
+service-name: IoT Firmware Defense
```

Comment:
What's the generated manager name for this service-name?

Reply:
https://github.com/azure-sdk/azure-sdk-for-java/blob/2f22feea91100b71a7fe62079d0b79640420d903/sdk/fist/azure-resourcemanager-fist/src/main/java/com/azure/resourcemanager/fist/IoTFirmwareDefenseManager.java

yeah, I thought I checked the casing there

Reply:
https://github.com/Azure/sdk-release-request/issues/4265#issuecomment-1631843232